container/Dockerfile.trtllm

```diff
@@ -441,72 +232,108 @@ ARG PACKAGING_VER=23.2
+COPY --from=framework /usr/local/lib/python${PYTHON_VERSION}/dist-packages/jinja2 /usr/local/lib/python${PYTHON_VERSION}/dist-packages/jinja2
+COPY --from=framework /usr/local/lib/python${PYTHON_VERSION}/dist-packages/jinja2-${JINJA2_VER}.dist-info /usr/local/lib/python${PYTHON_VERSION}/dist-packages/jinja2-${JINJA2_VER}.dist-info
+COPY --from=framework /usr/local/lib/python${PYTHON_VERSION}/dist-packages/sympy /usr/local/lib/python${PYTHON_VERSION}/dist-packages/sympy
+COPY --from=framework /usr/local/lib/python${PYTHON_VERSION}/dist-packages/sympy-${SYMPY_VER}.dist-info /usr/local/lib/python${PYTHON_VERSION}/dist-packages/sympy-${SYMPY_VER}.dist-info
+COPY --from=framework /usr/local/lib/python${PYTHON_VERSION}/dist-packages/flash_attn /usr/local/lib/python${PYTHON_VERSION}/dist-packages/flash_attn
+COPY --from=framework /usr/local/lib/python${PYTHON_VERSION}/dist-packages/flash_attn-${FLASH_ATTN_VER}.dist-info /usr/local/lib/python${PYTHON_VERSION}/dist-packages/flash_attn-${FLASH_ATTN_VER}.dist-info
+COPY --from=framework /usr/local/lib/python${PYTHON_VERSION}/dist-packages/flash_attn_2_cuda.cpython-*-*-linux-gnu.so /usr/local/lib/python${PYTHON_VERSION}/dist-packages/
+
+
+# Copy benchmarks, examples, and tests for CI
+COPY . /workspace/
+
+# Copy attribution files
 COPY ATTRIBUTION* LICENSE /workspace/
-
 # Copy launch banner
 RUN --mount=type=bind,source=./container/launch_message.txt,target=/workspace/launch_message.txt \
     sed '/^#\s/d' /workspace/launch_message.txt > ~/.launch_screen && \
-    echo "cat ~/.launch_screen" >> ~/.bashrc
+    echo "cat ~/.launch_screen" >> ~/.bashrc && \
+    echo "source $VIRTUAL_ENV/bin/activate" >> ~/.bashrc
+
+# Set PYTHONPATH to include workspace for benchmarks module, TODO: fix this to be more specific
+ENV PYTHONPATH=${DYNAMO_HOME}:${DYNAMO_HOME}/components/metrics/src:${DYNAMO_HOME}/components/frontend/src:${DYNAMO_HOME}/components/planner/src:${DYNAMO_HOME}/components/backends/trtllm/src
 
 ENTRYPOINT ["/opt/nvidia/nvidia_entrypoint.sh"]
 CMD []
+
+###########################################################
+########## Development (run.sh, runs as root user) ########
+###########################################################
+#
+# PURPOSE: Local development environment for use with run.sh (not Dev Container plug-in)
+#
+# This stage runs as root and provides:
+# - Development tools and utilities for local debugging
+# - Support for vscode/cursor development outside the Dev Container plug-in
+#
+# Use this stage if you need a full-featured development environment with extra tools,
+# but do not use it with the Dev Container plug-in.
+
+FROM runtime AS dev
+
+# Don't want ubuntu to be editable, just change uid and gid.
+ARG WORKSPACE_DIR=/workspace
+
+# Install utilities as root
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends  \
+    # Install utilities
+    nvtop \
+    wget \
+    tmux \
+    vim \
```

Comment:
If it is installed above it doesn't need to be here, note I would prefer it here to above though.